File: Modul 4/SOAL-4/PRAK404_2410817220015_Ashma_Salimah.py
while True : 
    print("Pilih program \n1. Penjumlahan \n2. Pengurangan \n3. Perkalian \n4. Pembagian \n5. Exit")
    x = int(input("Masukkan Pilihan : "))
    
    if 1 <= x <= 4 :
        y = float(input("Masukkan nilai pertama : "))
        z = float(input("Masukkan nilai kedua : "))
        
        if x == 1 :
            hasil_jumlah = y + z
            print(f"Hasil penjumlahan antara {y:.2f} dengan {z:.2f} adalah {hasil_jumlah:.2f}\n")
        elif x == 2 :
            hasil_kurang = y - z
            print(f"Hasil pengurangan antara {y:.2f} dengan {z:.2f} adalah {hasil_kurang:.2f}\n")
        elif x == 3 :
            hasil_kali = y * z
            print(f"Hasil perkalian antara {y:.2f} dengan {z:.2f} adalah {hasil_kali:.2f}\n")
        elif x == 4 :
            if z !=0 and y!=0 :
                hasil_bagi = y / z
                print(f"Hasil pembagian antara {y:.2f} dengan {z:.2f} adalah {hasil_bagi:.2f}\n")
            else :
                print("Error! = Pembagian tidak boleh 0!\n")

    elif x == 5 :
            print("Terimakasih, telah menggunakan kalkulator ASHMA SALIMAH \n")
            break

    else :
            print("Input anda salah, silahkan coba lagi\n")

File: Modul 4/SOAL-4/PRAK404_2410817220015_Ashma_Salimah.c
#include <stdio.h>
int main() {
    int x;
    float y, z;
    do {
        printf("Pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &x);
        if (x>=1 && x<=4) { printf("Masukkan nilai pertama : ");
            scanf("%f", &y);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &z); }
        if (x == 1) { float hasil_jumlah = y + z;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n\n", y, z, hasil_jumlah); }
        else if (x == 2) { float hasil_kurang = y - z;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n\n", y, z, hasil_kurang); }
        else if (x == 3) { float hasil_kali = y * z;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n\n", y, z, hasil_kali); }
        else if (x == 4) { float hasil_bagi = y / z;
            if (z != 0) { printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n\n", y, z, hasil_bagi);}
            else {printf("Error! = Pembagian tidak boleh 0!");} }
        else if (x == 5) { printf("Terimakasih, telah menggunakan kalkulator ASHMA SALIMAH \n\n"); }
        else { printf("Input anda salah, silahkan coba lagi \n\n"); }
    } while (x!=5);
    return 0;
}